test/unit-tests/mobile/devices-service.ts

```diff
@@ -437,6 +440,8 @@ describe("devicesService", () => {
 			it("does not fail when androidDeviceDiscovery startLookingForDevices fails", async () => {
 				(<any>androidDeviceDiscovery).startLookingForDevices = (): Promise<void> => { throw new Error("my error"); };
 				iOSDeviceDiscovery.emit("deviceFound", iOSDevice);
+				let hostInfo = testInjector.resolve("hostInfo");
+				hostInfo.isDarwin = true;
```

Comment:
The purpose of the test is to check if `startLookingForDevices` fails, which is in no way compromised by adding hostInfo.isDarwin. It's used to resolve an emulator service, and by that time the test has either failed or not.